backend/src/main/java/com/empire/World.java

```diff
@@ -82,6 +85,7 @@ public static World load(long gameId, int turn, DatastoreService service) throws
 	}
 
 	public static World fromJson(String json) {
+		log.warning(json);
```

Comment:
Why is this a "warning"?  Big JSON coming through?

Reply:
Oops! This was me debugging why I was getting JSON deserialization errors (see above), and I forgot to remove it.